frontend/Dockerstub

```diff
@@ -2,9 +2,8 @@ RUN apt-get update && apt-get install --yes nginx
 
 COPY --from=svelte --chown=root:root /home/node/frontend/dist /ofrak_gui
 
-ARG OFRAK_DIR=.
-COPY $OFRAK_DIR/frontend/nginx.conf /etc/nginx/sites-enabled/default
+COPY $PACKAGE_PATH/nginx.conf /etc/nginx/sites-enabled/default
 
-COPY $OFRAK_DIR/mkdocs.yml /mkdocs.yml
-COPY $OFRAK_DIR/docs /docs
-COPY $OFRAK_DIR/examples /examples
+COPY $PACKAGE_PATH/../mkdocs.yml /mkdocs.yml
+COPY $PACKAGE_PATH/../docs /docs
+COPY $PACKAGE_PATH/../examples /examples
```

Comment:
This seems like a bit of a hack to me, and the way we are doing it is causing these `COPY` directives to be done twice when both `frontend` and `tutorial` packages are included. Any reason not create an actual "`ofrak_docs`" (or some such) package that would include docs, examples, and mkdocs.yml and its own `Dockerstub` with these `COPY` instructions? Just a thought...

Reply:
I had a similar thought when refactoring this. I've made an issue for this, https://github.com/redballoonsecurity/ofrak/issues/426#issue-2142324456, as it is related to other documentation changes that make sense. Either way, I don't think this should block this PR.